### DISCUSSION TITLE
Billing & support problem -- I created a repo for a non-technical client many years ago

### DISCUSSION THREAD

**Discussion by dgaedcke** [community/community#50129] [New to GitHub]
We used their corp and there credit card to establish the account.  I've not worked for them in years but apparently that credit card is no longer working and I can't get Github support to remove the flag on my account.  Also surprising is that I can't seem to Sponsor open source projects while my account is in this state.  I've messaged support on 2 separate occasions but had no response or other follow-up to solve the problem!!

Who can I contact to solve this issue?

**Comment by neutrinomuon** [comment]
Hi, as I understand, you have already tried GitHub Contact Form: https://support.github.com/request under 'Account or billing issues' or request quickly or [open a support ticket](https://support.github.com/contact?source=subtitle&tags=rr-general-technical).

Maybe you can try to reach GitHub Twitter Support by tweeting to @GitHubHelp. They are known to be very responsive on social media, so this could be a quicker way to get a response.

I hope this helps

**Comment by Anmol-Baranwal** [comment]
I'm sorry to hear that you've been having trouble resolving this issue with GitHub support. Here are some steps you can take to try and resolve the problem:

1. Try reaching out to GitHub support again: It's possible that your previous messages got lost in the shuffle, or there was some other issue preventing them from getting back to you. Try reaching out to them again and be as specific as possible about your issue.

2. Look for alternative ways to contact GitHub support: If you're having trouble getting a response through the usual channels, try looking for alternative ways to get in touch. For example, you could try contacting them through social media or using a different email address.

3. Check the GitHub help center: GitHub has a comprehensive help center that may have the answers to your question. You can search for specific issues and see if there are any articles that address your problem.

4. Contact your former employer: If you used their corporate credit card to establish your GitHub account, they may be able to help you resolve the issue.

5. Consider creating a new GitHub account: If all else fails, you may need to create a new GitHub account using a different email address and payment method.

I hope this helps, and good luck resolving your issue with GitHub!

**Comment by queenofcorgis** [comment]
Hi there! Thanks for reaching out. The community is unable to assist in billing matters nor escalate Support tickets. However, if you share the ticket number with me I'm happy to ensure it went to the right place.

**Reply by dgaedcke** [reply]
#2071760

**Reply by queenofcorgis** [reply]
Hi there - your ticket is in the right place - it will be answered in the order it was received based on our team's resources. We appreciate your patience!
